src/ts/pages/main/toCart.ts

```diff
@@ -0,0 +1,31 @@
+import { countPrice } from './countPrice';
+
+export function addListener() {
+  const cardsBtn = document.querySelectorAll('.cards__btn');
+  cardsBtn.forEach((btn, index) => {
+    btn.addEventListener('click', (e) => {
+      const target = e.target as HTMLElement;
+      addToCardt(target, index);
+    });
+  });
+}
+
+function addToCardt(target: HTMLElement, index: number) {
+  if ((target.textContent as string).trim() === 'Add to cart') {
+    target.textContent = 'Drop from cart';
+    let arrayItems;
+    if (localStorage.getItem('basket') !== null) {
+      arrayItems = JSON.parse(localStorage.getItem('basket') || '[]');
+    } else {
+      arrayItems = [];
+    }
+    arrayItems.push(index + 1);
+    localStorage.setItem('basket', JSON.stringify(arrayItems));
+  } else {
+    target.textContent = 'Add to cart';
+    const arrayItems: number[] = JSON.parse(localStorage.getItem('basket') || '[]');
```

Comment:
Could all methods which work with localStorage be in a separate file? 
It will helps to faster change smth and support the code)